Gemfile

```diff
@@ -1,11 +1,16 @@
 source 'https://rubygems.org'
 
 # upstream
-gem 'fluentd', '~>1.9.1'
-gem 'activesupport', '~>6.0.2.1'
-gem 'fluent-plugin-prometheus', '~>1.7.0'
-gem 'fluent-plugin-multi-format-parser', '~>1.0.0'
-gem 'oj', '~>3.10.2'
+gem 'activesupport', '6.0.3.2'
+gem 'fluentd', '1.11.1'
+gem 'fluent-plugin-concat', '2.4.0'
+gem 'fluent-plugin-detect-exceptions', '0.0.13'
+gem 'fluent-plugin-elasticsearch', '4.1.0'
+gem 'fluent-plugin-kubernetes_metadata_filter', '2.5.2'
+gem 'fluent-plugin-multi-format-parser', '1.0.0'
+gem 'fluent-plugin-prometheus', '1.8.2'
+gem 'fluent-plugin-systemd', '1.0.2'
+gem 'oj', '3.10.7'
```

Comment:
We are not using half of those plugins, I cleaned them up in #20. Sadly, my commit message is non-existent and I don't remember the reason. Probably trying to reduce resources usage? I'd remove them, since we don't use them, but adding a comment for future updates from upstream

Reply:
I'm not sure I agree, a hundred MB saved isn't worth the overhead of deviating from upstream and having to check the versions/plugins everytime we update.

I say we leave them in, and don't spend time thinking about it.

Reply:
50M actually